src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1412,6 +1413,8 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
             bool is_depthwise = conv.get_primitive()->groups == input_layout.size.feature[0];
             bool first_conv = input_layout.size.feature[0] <= 4;
             if (((has_groups && !is_depthwise) || first_conv) &&
+            // Mingyu) isn't it necessary to change `output_layout.format --> preferred_format`?
```

Comment:
I think it is fine. It is used by the updating logic at the end of reorder_inputs.
It is from the PR https://github.com/openvinotoolkit/openvino/pull/9483.